src/styles/index.scss

```diff
@@ -2,2 +1,137 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__model {
+    margin-top: 40px;
+    width: 166px;
+    height: 36px;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 500;
+      padding-top: 0;
+      color: #060b35;
+    }
+  }
+
+  &__product-code {
+    align-self: start;
+    width: 98px;
+    height: 14px;
+    margin-top: 4px;
+
+    &-style {
+      white-space: nowrap;
+      padding-left: 17px;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+    }
+  }
+
+  &__grade {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-stars {
+      box-sizing: border-box;
+      width: 96px;
+      height: 16px;
+
+      &-list {
+        display: block;
+        list-style: none;
+        justify-content: space-between;
+        margin: 1px;
+
+        &:nth-child(n + 1) {
+          display: block;
+          background-image: url(../images/star.png);
+          width: 14px;
+          height: 14px;
+        }
+
+        &--5 {
+          display: none;
+        }
```

Comment:
Using `display: none;` to hide an element is not recommended if the element might be shown later. If this is for a dynamic feature like a rating system, consider toggling visibility with a class instead.